rest/model/generated_converters.go

```diff
@@ -11,10 +11,6 @@ func ArrstringArrstring(t []string) []string {
 	return m
 }
 
-func BoolBool(in bool) bool {
```

Comment:
What creates this? I'm not sure you can edit generated files.

Reply:
// Code generated by legacy codegen tool.
// Manual edits are allowed but please do not replicate this code style.